templates/zerver/left_sidebar.html

```diff
@@ -49,7 +49,7 @@
                 <a href="#streams">
                     <i id="streams_inline_cog" class='icon-vector-cog' data-toggle="tooltip" title="{{ _('Subscribe, add, or configure streams') }}"></i>
                 </a>
-                <a href=""><i id='streams_filter_icon' class='icon-vector-search' data-toggle="tooltip" title="{{ _('Filter streams list') }}"></i></a>
+                <a href=""><i id='streams_filter_icon' class='icon-vector-search' data-toggle="tooltip" title="{{ _('Filter streams list') }} (w)"></i></a>
```

Comment:
Maybe we should change it to "Filter streams" (w)?  The current thing line-wraps badly.